docs/gethue/content/en/posts/2021-07-26-zero-click-hue-importer-drag-and-drop-a-file-auto-create-a-table.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Zero Click Hue Importer -- Drag and drop a file, auto create a table
+author: Hue Team
+type: post
+date: 2021-07-26T00:00:00+00:00
+url: /blog/2021-07-26-zero-click-hue-importer-drag-and-drop-a-file-auto-create-a-table
+sf_thumbnail_type:
+  - none
+sf_thumbnail_link_type:
+  - link_to_post
+sf_detail_type:
+  - none
+sf_page_title:
+  - 1
+sf_page_title_style:
+  - standard
+sf_no_breadcrumbs:
+  - 1
+sf_page_title_bg:
+  - none
+sf_page_title_text_style:
+  - light
+sf_background_image_size:
+  - cover
+sf_social_sharing:
+  - 1
+sf_related_articles:
+  - 1
+sf_sidebar_config:
+  - left-sidebar
+sf_left_sidebar:
+  - Sidebar-2
+sf_right_sidebar:
+  - Sidebar-1
+sf_caption_position:
+  - caption-right
+sf_remove_promo_bar:
+  - 1
+ampforwp-amp-on-off:
+  - default
+categories:
+  - Version 4.11
+  - Development
+  - Query
+
+---
+
+In the last month, we introduced [creating a table from the local file](/blog/2021-05-26-improved-hue-importer-select-a-file-choose-a-dialect-create-a-table/) in importer. In that feature, if you want to create a table then you must have to follow the steps provided in that blog post but now only drag and drop a CSV file in the left assist, then it automatically creates the table for you.
+
+![Importer direct upload steps gif](https://cdn.gethue.com/uploads/2021/07/drag_and_drop_importer2.gif)
+
+
+Play with this feature in the latest Hue or at [demo.gethue.com](https://demo.gethue.com/hue/indexer/importer).  
+</br>
+</br>
+Any feedback or question? Feel free to comment here or on the [Forum](https://discourse.gethue.com/) and [quick start](https://docs.gethue.com/quickstart/) SQL querying!
```

Comment:
For feedback, questions, or suggestions, feel free to comment here or on the forum...